inst/stan/epinowcast.stan

```diff
@@ -76,6 +79,9 @@ parameters {
   vector<lower=0>[neff_sds] logmean_sd; // pooled modifiers to logmean
   vector<lower=0>[neff_sds] logsd_sd; // pooled modifiers to logsd
   vector<lower=0>[nrd_eff_sds] rd_eff_sd; // pooled modifiers to report date
+  real alpha_start[g]; // starting value for alpha
+  real<lower=0> alpha_sd; // standard deviation of the random walk increments
+  vector<offset=0, multiplier=alpha_sd>[t] alpha_epsilon[g]; // random walk increments, non-centered
```

Comment:
I am using the offset-multiplier style to declare non-centered parameterizations here...